scripts/diagnose-scheduler.sh

```
@@ -0,0 +1,88 @@
+    echo "   1. Check /tmp/scheduler.log for startup errors"
+    echo "   2. Manually run: bun run scheduler"
+    echo "   3. Check if required environment variables are set"
+fi
+
+if [ ! -f /tmp/scheduler.log ]; then
+    echo "⚠️  No scheduler log found. The scheduler may have never started."
+    echo "   Check the entrypoint script output during container startup."
+fi
+
+echo ""
+echo "=== END DIAGNOSTIC REPORT ===">
```

Shellcheck (0.10.0)

[error] Couldn't parse this redirection. Fix to allow more checks.

(SC1073)